### ISSUE TITLE
Placeholder options

### ISSUE THREAD

**Issue by TheRealLexii** [BentoBoxWorld/BSkyBlock#104]
When trying to set up featherboard for a server. It would be nice to have more placeholder options, such as island level, team members, island size etc. Pretty much most of what was used in Askyblock i believe. 

I have noticed there is a few placeholders already just waiting on the release of more if it is capable to do. 

Would also be nice to have a island size upgrade, not sure if this was a built in feature with Askyblock or another addon/plugin. Apologies if it was.

**Comment by TheRealLexii** [comment]
Also would be good to have a support discord if you dont already have one. Tried looking for one.

**Comment by BONNe** [comment]
Discord link: https://discordapp.com/invite/T5HxmvK
Placeholders in BentoBox 1.5: https://github.com/BentoBoxWorld/BentoBox/wiki/Placeholders
You can play with island range and island protection range to get size upgrade ability.

**Comment by TheRealLexii** [comment]
For %[gamemode] would this be bskyblock?

**Comment by BONNe** [comment]
yes
